.github/workflows/CI.yml

```diff
@@ -493,50 +451,68 @@ jobs:
             VERSION
           key: e2etest-release-${{ github.run_id }}
 
+      - name: restore cached rpms
+        uses: actions/cache@v2
+        with:
+          key: "cached_packages_${{ github.run_id }}"
+          path: $PACKAGING_ROOT
+
+      - name: Restore cached binaries
+        if: steps.e2etest-release.outputs.cache-hit != 'true'
+        uses: actions/cache@v2
+        with:
+          key: "cached_binaries_${{ github.run_id }}"
+          path: build
+
+      - name: Set up Docker Buildx
+        if: steps.e2etest-release.outputs.cache-hit != 'true'
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Public Integration Test ECR
+        if: steps.e2etest-release.outputs.cache-hit != 'true'
+        uses: docker/login-action@v1
+        with:
+          registry: public.ecr.aws
+          username: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
+          password: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
+        env:
+          AWS_REGION: us-east-1
+
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-access-key-id: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
           aws-region: us-west-2
 
-      - name: restore cached rpms
-        uses: actions/cache@v2
-        with:
-          key: "cached_packages_${{ github.run_id }}"
-          path: build/packages
-
       - name: Create SSM package
         run: |
-          ssm_pkg_version=`cat build/packages/VERSION`
+          ssm_pkg_version=`cat $PACKAGING_ROOT/VERSION`
```

Comment:
Should there also be a cache miss check here?

Reply:
IMO its not a good idea since we want to always create a SSM for later destroy and [you have filed a PR for this before ](https://github.com/aws-observability/aws-otel-collector/pull/758/files).

Reply:
Haha you're correct. Completely forgot about that.